src/test/java/com/marizueva/laboratory/hw4/voidpageobject/pages/ServiceDifferentElementPage.java

```diff
@@ -0,0 +1,115 @@
+package com.marizueva.laboratory.hw4.voidpageobject.pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ServiceDifferentElementPage extends CommonPageElements {
+
+    @FindBy(css = "input[type='checkbox'")
+    List<WebElement> checkBoxes;
+
+    @FindBy(css = "input[type='radio'")
+    List<WebElement> radioButtons;
+
+    @FindBy(css = "div.colors")
+    List<WebElement> dropdownMenu;
+
+    @FindBy(css = ".uui-side-bar.right-fix-panel")
+    WebElement rightPanel;
+
+    @FindBy(css = "div.checkbox-row > label.label-checkbox:nth-child(1)")
+    WebElement waterCheckbox;
+
+    @FindBy(css = "div.checkbox-row > label.label-checkbox:nth-child(3)")
+    WebElement windCheckbox;
```

Comment:
Checkboxes could be combined to the `List<WebElement>`

Reply:
fixed